tests/album-create.test.js

```diff
@@ -0,0 +1,49 @@
+// tests/album-create.js
```

Comment:
You dont need to include the file name at the top of each file as a comment, it just adds noise so should be removed